model/AI/Classes/AIPackage.md

```diff
@@ -0,0 +1,107 @@
+-------
```

Comment:
why is this additional line here? this might break parsing

Reply:
Thanks. Fixed.